pacman/PKGBUILD

```diff
@@ -5,6 +5,7 @@
 pkgname=pacman
 pkgver=5.0.1
 pkgrel=5
+epoch=1
```

Comment:
Git for Windows does not modify `pacman` directly. Therefore this change should be unnecessary.